services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/StatisticsToProtobufMapping.java

```diff
@@ -118,7 +118,7 @@ private LocalDate effectiveDateStringToLocalDate(String effectiveDate) {
             card = keyFigureCardFactory.createKeyFigureCard(stat, id);
             figureCardMap.put(id, Optional.of(card));
           } catch (MissingPropertyException ex) {
-            logger.warn(ex.getMessage());
+            logger.warn("[{}] {}", stat.getEffectiveDate(), ex.getMessage());
```

Comment:
@fredrb / @ioangut - what's the best error message? What helps most to identify root cause?

Reply:
After this log messages that will be logged as warnings, another log warning message will be clarifying them:       logger.warn("Statistics file will not be generated due to previous errors!");

Reply:
This one I think is in a good state now, because it will print what card, for what date, didn't have all the required properties